src/Expression.hs

```diff
@@ -13,31 +13,29 @@ module Expression (
     interpret, interpretInContext,
 
     -- * Convenient accessors
-    getBuildRoot, getContext, getPkgData, getPkgDataList, getOutputs, getInputs,
-    getInput, getOutput,
+    getBuildRoot, getContext, getOutputs, getInputs,
+    getInput, getOutput, getConfiguredCabalData,
 
     -- * Re-exports
     module Base,
     module Builder,
     module Context,
-    module GHC
     ) where
 
 import Base
-import Builder
+import {-# SOURCE #-} Builder
```

Comment:
Can you get rid of cyclic imports by adding `Builder.Type`?

Reply:
Hmm good question. I might give this a shot once I've done with the bigger problems.